### PR TITLE
[BEAM-2956] Make the DataflowRunner correctly report that it is using the Dataflow distribution.

### PR DESCRIPTION
**Pull request by lukecwik**
Note that this test will not pass until https://github.com/apache/beam/pull/3849 is merged
and the Apache Beam nightly 2.2.0-SNAPSHOT is updated.

### CODE REVIEW

**Comment by lukecwik** [comment]
R: @kennknowles

**Comment by lukecwik** [comment]
retest this please

**Comment by lukecwik** [comment]
R: @tgroh

File: maven-archetypes/examples-java8/src/main/resources/archetype-resources/src/main/java/common/WriteOneFilePerWindow.java [hunk 1]
```diff
@@ -17,21 +17,24 @@
  */
 package ${package}.common;
 
-import static com.google.common.base.Verify.verifyNotNull;
+    import static com.google.common.base.MoreObjects.firstNonNull;
```
Comment (tgroh) [inline maven-archetypes/examples-java8/src/main/resources/archetype-resources/src/main/java/common/WriteOneFilePerWindow.java]: This formatting seems wrong

Comment (lukecwik) [reply maven-archetypes/examples-java8/src/main/resources/archetype-resources/src/main/java/common/WriteOneFilePerWindow.java]: Copy and paste error, fixed.

**Pull request merged**